packages/inter-protocol/src/price/roundsManager.js

```diff
@@ -479,7 +479,7 @@ export const makeRoundsManagerKit = defineDurableFarClassKit(
             overrideValueOut === undefined
               ? lastValueOutForUnitIn // Use the latest value.
               : overrideValueOut; // Override the value.
-          if (valueOutForUnitIn === undefined) {
+          if (valueOutForUnitIn === null) {
```

Comment:
So we don't have test coverage for this? That makes me a little nervous. But I guess I can deal.

Also... `if (!valueOutForUnitIn) {` wouldn't be better?

Reply:
I hit this in `agops-vaults-smoketest.sh` so we have manual coverage. I'll consider adding an automated test. 

> Also... if (!valueOutForUnitIn) { wouldn't be better?

The looser condition would have prevented this bug but could potentially hide another.